gui-libs/wpebackend-fdo/wpebackend-fdo-1.14.3.ebuild

```diff
@@ -27,7 +27,7 @@ BDEPEND="
 "
 
 PATCHES=(
-	"${FILESDIR}"/${P}-c23-fix.patch
+	"${FILESDIR}"/wpebackend-fdo-1.14.3-c23-fix.patch
```

Comment:
and you can use "${FILESDIR}"/${PN}-1.14.3-c23-fix.patch for wpebackend-fdo-1.14.3.ebuild

JFYI: variables are descibed at https://devmanual.gentoo.org/ebuild-writing/variables/ ;)